HOW_TO_TEST.md

```diff
@@ -0,0 +1,65 @@
+# How to test
+
+Starting with CP-Ansible 5.5.0, we have included testing via the [Molecule](https://molecule.readthedocs.io/en/latest/) framework, and strongly advise it's usage before submitting a Pull Request.
+
+## Prerequisites
+
+1. Python3 installed with PIP
+2. Docker installed
+3. Install the Molecule and Docker libraries
+
+```pip install molecule docker```
+
+## Using Molecule
+
+The following is a list of the most common commands used with Molecule.
+
+### Running a role
+
+We have created a ```confluent.test``` role which contains all current scenarios for CP-Ansible under the ```molecule``` directory.  Each scenario named using the following format:
```

Comment:
Theres a good chance some molecule tests might pop up in more roles than the confluent.test role. You could say
Molecule allows for testing a role and will live inside a role's directory in a directory named "molecule". There are currently many integration tests inside the confluent.test role.